xdesign/plot.py

```diff
@@ -328,17 +328,23 @@ def _make_axis():
     return fig, axis
 
 
-def discrete_phantom(phantom, size, ratio=9, uniform=True,
-                     prop='linear_attenuation'):
+def discrete_phantom(phantom, psize, bounding_box=((0, 0), (1, 1)),
```

Comment:
Yes it was originally because I was trying to keep it consistent with the data format of `geometry.Point`, but I like the idea of column vectors as well. Thanks for the new PR